kustomize/commands/version/version.go

```diff
@@ -62,12 +63,24 @@ func (o *Options) Validate(_ []string) error {
 }
 
 func (o *Options) Run() error {
+	bi, ok := debug.ReadBuildInfo()
+	if !ok {
+		return fmt.Errorf("unable to read build information")
+	}
+
+	var version string
+	for _, dep := range bi.Deps {
+		if dep.Path == "sigs.k8s.io/kustomize/kustomize" {
+			version = dep.Version
+			break
+		}
+	}
 	switch o.Output {
 	case "":
 		if o.Short {
-			fmt.Fprintln(o.Writer, provenance.GetProvenance().Short())
+			fmt.Fprintln(o.Writer, version)
 		} else {
-			fmt.Fprintln(o.Writer, provenance.GetProvenance().Semver())
+			fmt.Fprintln(o.Writer, version)
```

Comment:
question: this will cause the output to be the same regardless of whether the `--short` flag was passed or not. Is that intentional?